setup.py

```diff
@@ -7,10 +7,11 @@
     author_email="forsyth2@llnl.gov, golaz1@llnl.gov, shaheen2@llnl.gov",
     description="Long term HPSS archiving software for E3SM",
     packages=find_packages(include=["zstash", "zstash.*"]),
+    python_requires=">=3.6",
     install_requires=[
-        "six==1.16.0",
-        "globus-sdk<4.0.0>=3.0.0",
-        "fair-research-login==0.2.6",
+        "six",
+        "globus-sdk>=2.0.0",
+        "fair-research-login",
```

Comment:
There seem to be questions about which of these is correct.  @forsyth2 and @lukaszlacinski, can you both weight in?

Reply:
I'm really not sure.

The changes to these values come from https://github.com/E3SM-Project/zstash/pull/201/files, which itself was from @lukaszlacinski's commit https://github.com/lukaszlacinski/zstash/commit/13c266029ef41742bb10f19cb6d8e1be99a4de58.

As @tomvothecoder mentioned in https://github.com/E3SM-Project/zstash/pull/201#issuecomment-1409061725: "The workaround was to constrain `globus-sdk >=2, <3` in `setup.py` in https://github.com/E3SM-Project/zstash/pull/188")

Reply:
@forsyth2, it looks to me like `setup.py` is correct, then and that @lukaszlacinski didn't know that he needed to keep `conda/meta.yaml` consistent.  But that also means that these constraints aren't in E3SM-Unified so hopefully we're still okay...